database/migrations/2021_09_17_000001_drop_attachments_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use EscolaLms\Core\Migrations\EscolaMigration;
+
+class DropAttachmentsTable extends EscolaMigration
```

Comment:
imho, just delete attachments migrations, we don't have any stable instances so far